### ISSUE TITLE
Custom ingressClass resource didn't create

### ISSUE THREAD

**Issue by falsedlah** [Kong/charts#698]
Use case: 

Two charts deployment:
      1. `kong` with the default setting 
      2. `kong-internal`: with `IngressClass: kong-internal`
```
...
ingressController:
      resources:
        requests:
          cpu: 50m
          memory: 64Mi
      ingressClass:  kong-internal
...
```
When I check the ingress class 
```
#kubectl get ingressclass -A

NAME   CONTROLLER                      PARAMETERS   AGE
alb    ingress.k8s.aws/alb             <none>       44d
kong   konghq.com/ingress-controller   <none>       252d
```

Expected: 
I expected to see two Kong ingressClass but I see only one which the default and I didn't see the custom one that I created

**Comment by rainest** [comment]
Do you have a particularly old version, something older than chart 2.8.0? `helm repo update` should pull down the latest one.

This was originally broken, but was fixed a while back: https://github.com/Kong/charts/commit/4f49605c16f66031a62c15e79d5784e2233cbd66

I don't see an obvious reason [the test to determine if we create the class](https://github.com/Kong/charts/blob/1e075db13b6500aba6603272a99969b99d55e3cb/charts/kong/templates/ingress-class.yaml#L1-L18) would fail: unless I'm misreading the truth table we only don't if it doesn't exist or if your cluster doesn't support IngressClass, neither of which appear true for your environment.

**Comment by falsedlah** [comment]
> Do you have a particularly old version, something older than chart 2.8.0? `helm repo update` should pull down the latest one.
> 
> This was originally broken, but was fixed a while back: [4f49605](https://github.com/Kong/charts/commit/4f49605c16f66031a62c15e79d5784e2233cbd66)
> 
> I don't see an obvious reason [the test to determine if we create the class](https://github.com/Kong/charts/blob/1e075db13b6500aba6603272a99969b99d55e3cb/charts/kong/templates/ingress-class.yaml#L1-L18) would fail: unless I'm misreading the truth table we only don't if it doesn't exist or if your cluster doesn't support IngressClass, neither of which appear true for your environment.

I was using old chart but when I updated it, it didn't work so I had to clean the existing resources and create them again. This did the trick and worked!
